sdk/core/core-http/review/core-http.api.md

```diff
@@ -599,6 +600,7 @@ export interface RequestOptionsBase {
     // (undocumented)
     [key: string]: any;
     abortSignal?: AbortSignalLike;
+    context?: Context;
```

Comment:
I think this should at least be `tracingContext` or `spanContext` or something, since otherwise it feels like we're describing some kind of HTTP context from the name alone

Reply:
I am not against this - the reason I did it (tell me how strongly you feel about this) is that structuring it this way means that this _exactly_ lines up with OperationTracingOptions. I could change it, but that seemed like a nice benefit (maybe doesn't matter).

Reply:
I guess I don't see the benefit, but maybe I'm missing where it becomes useful

Reply:
Renamed to `tracingContext` everywhere.